settings.yaml

```diff
@@ -4,14 +4,18 @@ min_num_yrs: 25
 last_train_year: 2003
 formula: y ~ bio2 + bio3 + bio5 + bio8 + bio9 + bio15 + bio16 + bio18 + ndvi_sum + ndvi_win + elevs
 vars: 
-  - bio2 
-  - bio3 
-  - bio5 
-  - bio8 
-  - bio9
-  - bio15 
-  - bio16 
-  - bio18 
+  - bio2 #Mean Diurnal Range (Mean of monthly (max temp - min temp))
```

Comment:
I like these comments.  Should the comments at the very bottom for 5,6,7 be there, or are they vestigial?